Tools/Scripts/webkitpy/benchmark_runner/data/plans/speedometer3.plan

```diff
@@ -0,0 +1,10 @@
+{
+    "timeout": 600,
+    "count": 4,
+    "remote_archive":"https://github.com/WebKit/Speedometer/archive/refs/heads/main.zip",
+    "github_source": "https://github.com/WebKit/WebKit/tree/d0bce574a12b3524176d177f737f324e1694dcfc/PerformanceTests/Speedometer",
+    "webserver_benchmark_patch": "data/patches/webserver/Speedometer3.patch",
+    "signpost_patch": "data/patches/signposts/Speedometer3.patch",
+    "entry_point": "index.html",
+    "output_file": "speedometer3.result"
```

Comment:
It would be worth running `python3 Tools/Scripts/webkitpy/benchmark_runner/plan_file_optimizer.py Tools/Scripts/webkitpy/benchmark_runner/data/plans/speedometer3.plan` to get `github_subtree` generated so that we can avoid GitHub API rate limit.

Reply:
Ryosuke pointed out `"remote_archive":"https://github.com/WebKit/Speedometer/archive/refs/heads/main.zip",` is from a separate repo, which means we don't need `github_subtree`.